sale_order_type/README.rst

```diff
@@ -35,6 +35,17 @@ You can see sale types as lines of business.
 You are able to select a sales order type by partner so that when you add a
 partner to a sales order it will get the related info to it.
 
+Rules can also be associated with sale order types.
+
+Inside each rule, you can select any number of products and/or product categories.
+
```

Comment:
Case 1 

- 1 sale order with 2 product
- Each of that 2 products are in 2 different rules of 2 different sale_order_type

Product 1 -> Sale order type 1
Product 2 -> Sale order type 2

Sale Order TEST has product 1 +  product 2

Which is the result?